algorithms/C/searching/linearsearch.c

```diff
@@ -0,0 +1,67 @@
+// linear search
+#include <stdio.h>
+#include <stdlib.h>
+//function prototype
+int linearSearch(int a[], int query, int size);
+
+int main()
+{
+   int *data,i,temp,searchQuery,size;
+   scanf("%d",&size);
```

Comment:
please add the UI here 
Hint: printf("Enter size: ");
So the user can understand that what to do